diff --git a/node_modules/@electron-forge/core/dist/api/make.js b/node_modules/@electron-forge/core/dist/api/make.js
index e17e0e2..c4d8449 100644
--- a/node_modules/@electron-forge/core/dist/api/make.js
+++ b/node_modules/@electron-forge/core/dist/api/make.js
@@ -189,7 +189,7 @@ var _default = async ({
            *   * Change the entire API of maker from a single constructor to
            *     providing a MakerFactory
            */
-          maker.prepareConfig(targetArch);
+          await Promise.resolve(maker.prepareConfig(targetArch));
           const artifacts = await maker.make({
             appName,
             forgeConfig,
