diff --git a/node_modules/electron-devtools-installer/dist/downloadChromeExtension.js b/node_modules/electron-devtools-installer/dist/downloadChromeExtension.js
index 4a0ef2a..ef97420 100644
--- a/node_modules/electron-devtools-installer/dist/downloadChromeExtension.js
+++ b/node_modules/electron-devtools-installer/dist/downloadChromeExtension.js
@@ -16,7 +16,7 @@ const downloadChromeExtension = (chromeStoreID, forceDownload, attempts = 5) =>
             if (fs.existsSync(extensionFolder)) {
                 rimraf.sync(extensionFolder);
             }
-            const fileURL = `https://clients2.google.com/service/update2/crx?response=redirect&acceptformat=crx2,crx3&x=id%3D${chromeStoreID}%26uc&prodversion=32`; // eslint-disable-line
+            const fileURL = `https://clients2.google.com/service/update2/crx?response=redirect&acceptformat=crx2,crx3&x=id%3D${chromeStoreID}%26uc&prodversion=130.0.6723.137`; // eslint-disable-line
             const filePath = path.resolve(`${extensionFolder}.crx`);
             utils_1.downloadFile(fileURL, filePath)
                 .then(() => {
